currency_rate_update/models/res_currency_rate_provider.py

```diff
@@ -285,11 +285,16 @@ def _scheduled_update(self):
 
         _logger.info('Scheduled currency rates update complete.')
 
-    @api.model
-    def _get_supported_currencies(self, service):
+    @api.multi
+    def _get_supported_currencies(self):  # pragma: no cover
+        self.ensure_one()
+
```

Comment:
Why adding this ugly empty lines inside a method?

Reply:
updated